web/packages/teleterm/src/ui/Vnet/vnetContext.tsx

```diff
@@ -465,6 +487,28 @@ export const VnetContextProvider: FC<
     ]
   );
 
+  const autoConfigureSSH = useCallback(async (): Promise<void> => {
+    await vnet.autoConfigureSSH({});
+    // Refresh the service info attempt because SSH is now configured.
+    refreshServiceInfoAttempt();
+  }, [vnet, refreshServiceInfoAttempt]);
+  const openSSHConfigurationModal = useCallback(
+    ({ vnetSSHConfigPath, host, onSuccess }) => {
```

Comment:
You can do something like this here, otherwise these args on line 496 are untyped in the context of this function.

```tsx
  const openSSHConfigurationModal: VnetContext['openSSHConfigurationModal'] =
```